pkgs/development/ocaml-modules/reason-native/default.nix

```diff
@@ -1,21 +1,21 @@
-{ newScope, lib, fetchFromGitHub, callPackage, buildDunePackage, atdgen, junit, qcheck-core, re, reason, reason-native }:
+{ newScope, lib, fetchFromGitHub, callPackage, buildDunePackage, atdgen, junit, qcheck-core, re, reason, reason-native, fetchpatch }:
 
 let
   generic = (somePath:
     let
       prepkg = import somePath {
-        inherit callPackage cli buildDunePackage atdgen junit qcheck-core re reason;
+        inherit callPackage cli buildDunePackage atdgen junit qcheck-core re reason fetchpatch;
         inherit (reason-native) console file-context-printer fp pastel rely;
       };
     in
       buildDunePackage
         ({
-          version = "2021-16-16-aec0ac6";
+          version = "2022-08-31-a0ddab6";
           src = fetchFromGitHub {
             owner = "reasonml";
             repo = "reason-native";
-            rev = "aec0ac681be7211b4d092262281689c46deb63e1";
-            sha256 = "sha256-QoyI50MBY3RJBmM1y90n7oXrLmHe0CQxKojv+7YbegE=";
+            rev = "a0ddab6ab25237961e32d8732b0a222ec2372d4a";
+            sha256 = "sha256-s2N5OFTwIbKXcv05gQRaBMCHO1Mj563yhryPeo8jMh8=";
           };
           useDune2 = true;
```

Comment:
```suggestion
          duneVersion = "3";
```
(not tested)

Reply:
I thought to do this here https://github.com/NixOS/nixpkgs/pull/223751 it is unrelated

Reply:
At least please get rid of the (deprecated) `useDune2` attribute.

Reply:
I see other left over useDune2 https://github.com/ulrikstrid/nixpkgs/blob/eeeb2e29997a7b43efe0535e0d69bb7636b3bfa5/pkgs/development/ocaml-modules/reason-native/default.nix#L20 not changed in the other PR. It does build if I change all `useDune2 = true;` to `duneVersion = "3";` so it can be done in the other PR. I will do it here if the other PR is merged before this one.

Reply:
Use then: `duneVersion = "2";`

Reply:
Ok I have done cleanup but ofborg tag this 0 Darwin rebuild. Does it have cached the very old version of that package build ?